common/model/pages.js

```diff
@@ -8,6 +8,9 @@ export type Page = {|
   promo: ?ImagePromo,
   body: any[],
   datePublished: ?Date,
+  // This is just for now, we will be implementing a proper site tagging
```

Comment:
Could this have a `TODO:` or `FIXME:` for ease of searching for stuff we want to update?

Reply:
Quite liking the syntax of `TODO ({label})` syntax.

Reply:
I'll need to configure my 'Find TODOs' plugin accordingly (think it only searches for `TODO:` and `FIXME:` by default)

Reply:
ah, okay, I'll do it differently.
How about `TODO: (drupal migration) blah blah`?